pkg/webhook/webhook.go

```diff
@@ -91,6 +99,19 @@ func New(
 		cert:              cert,
 		serviceRef:        serviceRef,
 		sparkJobNamespace: jobNamespace,
+		failurePolicy:     failurePolicy,
+	}
+
+	if webhookNamespaceSelector != "" {
+		kv := strings.SplitN(webhookNamespaceSelector, "=", 2)
```

Comment:
Should we allow more multi-selector, e.g., `key1=val1,key2=val2`?

Reply:
I think it's best to keep this as simple until given a reason otherwise. If filtering on multiple values is needed, users can use e.g. `foo=sparkoperatorv123` instead of `sparkoperator=yes,sparkoperatorversion=123`.